caluma/extensions/settings.py

```diff
@@ -69,6 +69,12 @@
     "additional-data-form",
 ]
 
+settings.REVISION_QUESTIONS = {
+    "review-document": ["priorisierung-der-antrage-kommentar"],
+    "decision-and-credit": ["gesprochener-rahmenkredit"],
+    "define-amount": ["define-amount-amount", "todo-remark"],
```

Comment:
`todo-remark`?  You can just define a slug.

Reply:
Changed to `define-amount-remark`